docs/Actions/send-email.md

```diff
@@ -0,0 +1,531 @@
+| Format                        | The format of the email body                                                                                                                                                                                                                                              | No              | `Html (And replace newline with <br />)` |
+| DNN email template to use.    | Choose from a predefined email from the system                                                                                                                                                                                                                            | No              | Unset                                    |
+| Body                          | The email template you want to send                                                                                                                                                                                                                                       | Yes             | `empty string`                           |
+| Headers                       | Append custom headers to the email.                                                                                                                                                                                                                                       | Yes             | Unset                                    |
+| Attach Portal Files           | Pick a file that you want to send as attachment via email                                                                                                                                                                                                                 | No              | Unset                                    |
+| Attach Field #                | Form only parameters, provide the ability to add up to 5 Single File Uploads fields to be send as attachments to the email                                                                                                                                                | No              | Unset                                    |
+| Attach from Token             | This lets you compute file path from token. It must be relative to portal home folder. Multiple items must be separated by semicolons.                                                                                                                                    | Yes             | `empty string`                           |
+| Ignore Errors                 | If errors are ignored, the submission of the form continues even when there is an error sending the email. Otherwise, an error message will be displayed to the user. Note that if the email is being sent async or is placed in a pickup folder, errors won't be caught. | No              | No                                       |

```

Comment:
Is this specific to the form? Or does it affect any action list/workflow?

Reply:
> Attach Field # 
Will only appear when the action is in the Form.
Those Parameters will not appear in other action lists of other modules including workflows.